### ISSUE TITLE
Possibility to disable Task Completed

### ISSUE THREAD

**Issue by bigben3333** [saoudrizwan/claude-dev#189]
Hello,

In the configuration of the plugin, is it possible to add an option to disable "Task Completed" ?
Most of the time it consume time & credit for nothing :)

Thx

**Comment by SamPink** [comment]
I agree with this tbh

**Comment by CiberNin** [comment]
Maybe better to allow claude to prompt followup questions on completion.

**Comment by saoudrizwan** [comment]
I agree, it's so unnecessarily verbose sometimes. The point of task completion is it's usually the point at which claude will run a command to show his results, or give you details about next steps. I actually did some test runs without task completion and I found claude had a hard time figuring out when he was even done with a task and tended to go on side quests like modifying the same file over and over again thinking it could be improved each time. I found the task completion tool was a good way for it to ask itself, "okay am I done now?", so there's more of a point to it than just letting you know it's done.
